build-aux/main.mk

```diff
@@ -193,7 +193,7 @@ format: $(tools/golangci-lint) $(tools/protolint) ## (QA) Automatically fix lint
 
 .PHONY: check
 check: $(tools/ko) $(tools/helm) ## (QA) Run the test suite
-	TELEPRESENCE_MAX_LOGFILES=300 go test -timeout=18m ./...
+	TELEPRESENCE_MAX_LOGFILES=300 SYSTEMA_HOST=127.0.0.1 TELEPRESENCE_LOGIN_DOMAIN=127.0.0.1 go test -timeout=18m ./...
```

Comment:
Add a comment that explains why these are set to localhost and add `SCOUT_HOST` to the list to ensure that broken reporting doesn't mess things up.

Reply:
Hmm, I see that `SCOUHT_HOST` is only used by `scout.py` so what's needed instead is to ensure that `https://metriton.datawire.io/scout` is redirected to localhost.